slides/src/content/unit-testing.md

```diff
@@ -248,70 +231,30 @@ mockTodoService = {
 - Using the test fixture, Angular will give us the instance it created
 
 ---
-<!-- .slide: id="unit-testing-providing-mock-dependencies" -->
-## Providing Mock Dependencies
+<!-- .slide: id="unit-testing-configure-the-test-module" -->
+## Configuring the Test Module
 
-- We can declare `mockTodoService` as we did before and have Angular inject it for us
+- We can declare `mockToDoService` as we did before and have Angular inject it for us
+- TestBed will also instantiate the `AppComponent` and inject the dependency for us, but we must add it to the `declarations` array.
 
 #####_src/app/app.component.spec.ts_
 ```ts
 TestBed.configureTestingModule({
-  declarations: [ AppComponent ]
+  declarations: [ AppComponent ],
   providers:[
-    { provide: TodoService, useValue: mockTodoService }
+    { provide: ToDoService, useValue: mockToDoService }
   ]
 })
 ```
 
 ---
-<!-- .slide: id="unit-testing-spying-on-real-service-providers" -->
-## Spying on Real Service Providers
-
-- Alternatively, we can have Angular create the real service
-- And mock specific methods through a Jasmine *spy*
-  - The spy will ensure we don't actually call the real method
-- Always obtain injected dependencies from `TestBed`
-  so that the spy is set on the instance that is injected and not the object we created
-
-#####_src/app/app.component.spec.ts_
-```ts
-TestBed.configureTestingModule({
-  declarations: [ AppComponent ]
-  providers:[ TodoService ]
-});
-
-const todoService = TestBed.get(TodoService);
-spyOn(todoService, 'itemCount').and.callFake(() => 3);
-```
-
-<!--
-FIXME: check the example above
-1. Do we still call `compileComponents` on `TestBed`?
-2. Is the syntax right for `callFake`?
--->
-
----
-<!-- .slide: id="unit-testing-importing-real-application-modules" -->
-## Importing Real Application Modules
-
-- We can import a real module that declares the component and provides the service it depends on
-
-```ts
-TestBed.configureTestingModule({
-  imports: [ AppModule ]
-})
-```
-
-- Inject actual services or importing real modules often works...
-- ...but sometimes Angular is not able to resolve all of the required dependencies
-- We have more control when we mock things...
-- ...but if mocks get too complicated, they can be expensive to maintain
 
----
 <!-- .slide: id="unit-testing-testing-components-using-componentfixture" -->
 ## Testing Components Using ComponentFixture
 
-Here is how we obtain the instance from the fixture:
+- `ComponentFixture` is an testing utility that gives us access and control over the things we are testing.
+- Here is how we obtain the instance of our declared component from the fixture.
```

Comment:
Rather than "Here is how we...", prefer "We obtain X by..." and be explicit - "by referencing in providers" ? or...?